.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -182,6 +182,25 @@ jobs:
           if-no-files-found: error
           retention-days: 7
 
+  run-aleph-client-subxt-codegen-check:
+    needs: [build-test-docker]
+    name: Checks if runtime file in aleph-client is up-to-date
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Run one node in the background
+        uses: ./.github/actions/run-e2e-test
+        with:
+          node-count: 1
+          min-validator-count: 1
+
+      - name: check if runtime metadata matches
+        run: |
+          cd aleph-client/
+          docker build --tag subxt:latest -f docker/Dockerfile .
```

Comment:
We should use more descriptive tag -- this one makes it sound like the docker image _is_ subxt. Maybe `subxt-client-integration`?

Reply:
Instead of building it every time we should use pre-built container.

Reply:
Both done